src/main/java/mate/academy/bookshop/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.bookshop.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookshop.validation.PasswordFieldMatch;
+import org.hibernate.validator.constraints.Length;
+
+@PasswordFieldMatch(message = "{register.repeatPassword.mismatch}")
+@Data
+public class UserRegistrationRequestDto {
+    private Long id;
+    @Email(message = "{register.email.invalid}")
```

Comment:
This annotation doesn't validate the length of input, so additionally to this annotation validate the input size  too.